js/data/bucket.js

```diff
@@ -65,8 +61,8 @@ function Bucket(options) {
     this.zoom = options.zoom;
     this.overscaling = options.overscaling;
     this.layer = options.layer;
+    this.childLayers = options.childLayers;
```

Comment:
I find the term `refLayer` confusing because it can refer either to a layer _with_ a `ref` property or one _referred to by_ a `ref` property. 

I have taken to calling layers _with_ a `ref` property "ref children" and layers _referred to by_ a `ref` property "ref parents".